File: app/src/main/java/com/scottmangiapane/cardclicks/HelpActivity.java
package com.scottmangiapane.cardclicks;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.activity.EdgeToEdge;
import androidx.activity.OnBackPressedCallback;
import androidx.appcompat.app.AppCompatActivity;

import com.scottmangiapane.cardclicks.game.Game;

public class HelpActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_help);

        findViewById(R.id.button_refresh).setOnClickListener(view -> refresh());
        findViewById(R.id.button_return_to_menu).setOnClickListener(v -> {
            finish();
            overridePendingTransition(R.transition.slide_left_1, R.transition.slide_left_2);
        });
        refresh();

        getOnBackPressedDispatcher().addCallback(this, new OnBackPressedCallback(true) {
            @Override
            public void handleOnBackPressed() {
                finish();
                overridePendingTransition(R.transition.slide_left_1, R.transition.slide_left_2);
            }
        });
    }

    private void refresh() {
        Game game = new Game(3);
        ((ImageView) findViewById(R.id.help_card_1)).setImageResource(game.get(0).drawable);
        ((ImageView) findViewById(R.id.help_card_2)).setImageResource(game.get(1).drawable);
        ((ImageView) findViewById(R.id.help_card_3)).setImageResource(game.get(2).drawable);
    }
}

File: app/src/main/java/com/scottmangiapane/cardclicks/HelpActivity.kt
package com.scottmangiapane.cardclicks

import android.os.Bundle
import android.view.View
import android.widget.ImageView
import androidx.activity.OnBackPressedCallback
import androidx.appcompat.app.AppCompatActivity
import com.scottmangiapane.cardclicks.game.Game

class HelpActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_help)
        findViewById<View>(R.id.button_refresh).setOnClickListener { refresh() }
        findViewById<View>(R.id.button_return_to_menu).setOnClickListener { finish() }
        refresh()
        onBackPressedDispatcher.addCallback(this, object : OnBackPressedCallback(true) {
            override fun handleOnBackPressed() {
                finish()
            }
        })
    }

    private fun refresh() {
        val game = Game(3)
        (findViewById<View>(R.id.help_card_1) as ImageView).setImageResource(
            game[0].drawable
        )
        (findViewById<View>(R.id.help_card_2) as ImageView).setImageResource(
            game[1].drawable
        )
        (findViewById<View>(R.id.help_card_3) as ImageView).setImageResource(
            game[2].drawable
        )
    }
}
